safety_module/src/lib.rs

```diff
@@ -232,7 +205,7 @@ pub trait SafetyModule {
             .set_special_roles(
                 &self.blockchain().get_sc_address(),
                 &token,
-                (&roles[..]).into_iter().cloned(),
+                (&roles[..]).iter().cloned(),
```

Comment:
clippy suggests:
`roles.iter().cloned()`

Reply:
Will fix both in the next PR.